import_package.md

```diff
@@ -0,0 +1,280 @@
+# Import package
+
+* An import package is a .zip file containing a **manifest.yaml** that describes additional 
+  contents and actions that should be carried out when processing the package.
+* A valid import package must contain all resource files that are referenced in the 
+  **manifest.yaml**.
+
+## Meta-data
+
+* `apiVersion` is set to `v1beta1` 
+
+## Tasks
+
+* `tasks`: is a list of api actions and resource tasks.
```

Comment:
```suggestion
* `tasks`: list of api actions and resource tasks.
```
Does not need to be a full sentence.

Could be:
* actions: list of api and resource actions